torch/nested/_internal/nested_int.py

```diff
@@ -0,0 +1,339 @@
+import torch
```

Comment:
Don't know if we actually want Python nested int due to eager slowness, but its easy to prototype with this.